e2e/cypress/integration/messaging/header_spec.js

```diff
@@ -36,7 +36,7 @@ describe('Header', () => {
         updateAndVerifyChannelHeader('>', 'newheader'.repeat(20));
     });
 
-    it('CS14730 - Channel Header: Markdown quote', () => {
+    it('MM-T881_1 - Header: Markdown quote', () => {
```

Comment:
I think this needs a follow up PR since the test case has 2 steps.

Reply:
Yeah, follow up on the second step in a follow up PR.
Thanks Saturn!